Assignments/Program.cs

```diff
@@ -1 +1,116 @@
-﻿Console.WriteLine("Hello, World!");
+﻿// ---------------------------------------------------------------------------------------
+// T2 - NUMBER TO WORDS AND ROMAN NUMERALS CONVERTER 
+// ---------------------------------------------------------------------------------------
+
+using System;
+using System.Globalization;
+using static System.ConsoleColor;
+
+namespace Spark;
+
+public class T2 {
+   public static void Main () {
+      List<int> wTestCases = new () { 0, 1, 4, 9, 10, 11, 16, 25, 36, 49, 50, 81, 100, 109, 125, 216, 343, 729, 1000, 12001, 66000, 123450, 1000909, 987654321, 100000001 };
+      List<int> rTestCases = new () { 1, 4, 5, 9, 10, 47, 94, 100, 345, 400, 499, 500, 900, 998, 1000, 1500, 1729, 1888, 3495, 3913, 4000, 4096, 4509, 4761, 4999 };
+      Console.WriteLine ("\x1B[4m" + "Numbers to Words:" + "\x1B[0m");
+      for (int i = 0; i < wTestCases.Count; i++) {

```

Comment:
Both the for loops have similar code. Refactor it to a method takes specific parameters to handle both cases.
     for (int i = 0; i < wTestCases.Count; i++) {
         var item = wTestCases[i];
         Console.Write ($"{i + 1,2}) {item,9} ==> ");
         WriteInYellow (ToWords (item));
         Console.ResetColor ();
      }
      Console.WriteLine ("\n\x1B[4m" + "Numbers to Romans:" + "\x1B[0m");
      for (int i = 0; i < rTestCases.Count; i++) {
         var item = rTestCases[i];
         Console.Write ($"{i + 1,2}) {item,4} ==> ");
         WriteInYellow (ToRomans (item));
         Console.ResetColor ();
      }